code/game/objects/items/weapons/storage/wallets.dm

```diff
@@ -30,7 +32,8 @@
 		/obj/item/device/encryptionkey,
 		/obj/item/fluff,
 		/obj/item/storage/business_card_holder,
-		/obj/item/sample
+		/obj/item/sample,
+		/obj/item/key
```

Comment:
This should be mentioned in the changelog

Reply:
aye aye captain. working on it.

Reply:
addressed.